src/protocol_nmea.cpp

```diff
@@ -1386,17 +1403,17 @@ int nmea_gsv_group(char a[], int aSize, int &offset, gps_sat_t &gsat, gps_sig_t
 				for (uint32_t j=0; j<=gsat.numSats; j++)
 				{
 					gps_sat_sv_t &sat = gsat.sat[j];
-					if (sat.gnssId == sig.gnssId &&
-						sat.svId == sig.svId)
+					if ((sat.gnssId == sig.gnssId) && (sat.svId == sig.svId)
+                        && (sat.elev >= 0)) // TODO: We should probably remove this, implement a parameter in the GPX flash config, and configure on the CXD5610.
```

Comment:
You switched the elevation print below to use nmea_print_i32() which will correctly display negative elevation angles.   But then you limit the elevation angles to only non-negative angles.  Are we sure we don't want to print negative angles?